tests/src/dns_test.cpp

```diff
@@ -551,3 +551,54 @@ TEST_F(DNSTest, SOARecordSerialize) {
     EXPECT_EQ(0x8ad71928U, r2.expire());
     EXPECT_EQ(0x1ad92871U, r2.minimum_ttl());
 }
+
+TEST_F(DNSTest, BadLabelSize) {
+    constexpr uint8_t header[] = {
```

Comment:
I think visual studio isn't liking this. Not sure if it's because of `constexpr`, this looks legal but the compiler may be a bit outdated.

Reply:
yeah seems like a subset of the compilers don't recognize constexpr yet; changed those to const...